views/partials/header.ejs

```diff
@@ -49,6 +49,7 @@ a {
         
         <!-- <img src="../../assets/ram.png" width="65px" height="50px"> -->
         <div class="flex nav">
+				<a href="/">Home</a>
```

Comment:
The title of the website, if clicked, will take you home. Do you think this is still necessary (serious question)